### ISSUE TITLE
Color contrast and "Resolutions" in the Minutes template

### ISSUE THREAD

**Issue by johnfoliot** [w3c/scribe2#8]
Currently, when a Resolution is posted to the IRC minutes, it renders in the minutes template with an orange background. 

WCAG 2.x mandates minimum color contrast requirements that our templates should be supporting. It is worth noting that Orange is extremely difficult to maintain a color contrast with. (I propose the easiest resolution is to change the orange to perhaps yellow, or a significantly paler shade of orange.)

### [Success Criterion 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum)
The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:

**Large Text**
Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;

**Incidental**
Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

**Logotypes**
Text that is part of a logo or brand name has no contrast requirement.

(A useful resource for this: [https://contrastchecker.com](https://contrastchecker.com))

**Comment by bert-github** [comment]
I changed the colors to black on #FD8 (a pale orange). The color contrast is now 16:1.

**Comment by johnfoliot** [comment]
Thank you Bert!
